docs/architecture/translations.rst

```diff
@@ -0,0 +1,84 @@
+Model Translations
+==================
+
+.. note::
+
+    The Model Translations are in the early version. They are fully functional, however, translations can be created only via custom code.
```

Comment:
"Custom code" sounds a bit vague. I'd say something like: "At this stage, model translations are only accessible from the Python code. GraphQL API and UI views will be added in the future releases."